README.md

```diff
@@ -96,6 +97,45 @@ If you plan to make contributions to OpenWhisk, we recommend either a Mac or Ubu
 * [Setup Mac for OpenWhisk](tools/macos/README.md)
 * [Setup Ubuntu for OpenWhisk](tools/ubuntu-setup/README.md)
 
+### Lean Setup
```

Comment:
Instead of this section, let's move the details into the configuration choices for each of the deployment options.